luma/oled/device/color.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018 Richard Hull and contributors
+# See LICENSE.rst for details.
+
+"""
+Base class for SSD13xx color devices
+
+.. versionadded:: 3.0.0
+"""
+
+from abc import abstractmethod, ABCMeta
+
+from luma.core.device import device
+import luma.core.error
+import luma.core.framebuffer
+import luma.oled.const
+
+
+class color_device(device):
+    __metaclass__ = ABCMeta
+
+    def __init__(self, serial_interface, width, height, rotate, framebuffer, **kwargs):
+        super(color_device, self).__init__(luma.oled.const.common, serial_interface)
+        self.capabilities(width, height, rotate, mode="RGB")
+        self.framebuffer = getattr(luma.core.framebuffer, framebuffer)(self)
+
+        if (width, height) not in self._supported_dimensions():
+            raise luma.core.error.DeviceDisplayModeError(
+                "Unsupported display mode: {0} x {1}".format(width, height))
+
+        self._init_sequence()
+        self.contrast(0xFF)
+        self.clear()
+        self.show()
+
+    @abstractmethod
+    def _supported_dimensions(self):
+        """
+        Enumerates the screen resolutions that the specific device supports, as
+        a list of tuples; e.g.: [(96, 64), (96, 32), (96, 16)]
```

Comment:
should be wrapped, e.g.
```
``[(96, 64), (96, 32), (96, 16)]``
```

Reply:
ping @rm-hull

Reply:
This was done ages ago - see https://github.com/rm-hull/luma.oled/pull/203/files#diff-12a7127a588670816142a0e5d0a15c1cR40 - the snippet above is showing as 'outdated'